src/AltinnCore/Runtime/appsettings.json

```diff
@@ -35,13 +35,13 @@
     "AltinnStudioEndpoint": "http://altinn3.no/"
   },
   "PlatformSettings": {
-    "ApiStorageEndpoint": "http://localhost:5010/storage/api/v1/",
+    "ApiStorageEndpoint": "http://localhost:5010/storage/api/v1",
     "ApiStorageEndpointHost": "localhost:5010",
-    "ApiRegisterEndpoint": "http://localhost:5020/register/api/v1/",
+    "ApiRegisterEndpoint": "http://localhost:5020/register/api/v1",
     "ApiRegisterEndpointHost": "localhost:5020",
     "ApiProfileEndpoint": "http://localhost:5020/profile/api/v1/",
     "ApiProfileEndpointHost": "localhost:5030",
-    "ApiAuthenticationEndpoint": "http://localhost:5040/authentication/api/v1/",
+    "ApiAuthenticationEndpoint": "http://localhost:5040/authentication/api/v1",
```

Comment:
Then the usage in account controller must be updated with a /

Reply:
Updated account controller.. it was merged in during development, so it got under my radar.. All other places the endpoints are used have been checked and updated if needed.